### PR TITLE
fix: single value legend (DHIS2-8348)

### PR DESCRIPTION
**Pull request by martinkrulltott**
Allows Single Value to change text color based on the first available legend.  https://jira.dhis2.org/browse/DHIS2-8348

image

Is linked to https://github.com/dhis2/data-visualizer-app/pull/722

### CODE REVIEW

**Review by edoardo — APPROVED** [review]
LGTM

File: src/visualizations/config/generators/dhis/singleValue.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
+import { getColorByValueFromLegendSet } from "../../../../modules/legends"
```
Comment (janhenrikoverland) [inline src/visualizations/config/generators/dhis/singleValue.js]: How did you get double quotes?

Comment (janhenrikoverland) [inline src/visualizations/config/generators/dhis/singleValue.js]: Double quotes

File: src/visualizations/config/generators/dhis/singleValue.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+
 const svgNS = 'http://www.w3.org/2000/svg'
 
-const generateValueSVG = (value, y) => {
+const generateValueSVG = (value, legendSet, y) => {
     const textSize = 300
+    const defaultFillColor = "#000000"
```
Comment (janhenrikoverland) [inline src/visualizations/config/generators/dhis/singleValue.js]: Double quotes

**Pull request merged**